Deceive/StartupHandler.cs

```diff
@@ -13,15 +13,9 @@
 
 namespace Deceive
 {
-    class MainClass
+    class StartupHandler
```

Comment:
Renamed class MainClass to class StartupHandler, as file name was Program.cs too.